src/Microsoft.AspNetCore.Blazor.Browser.JS/package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-6OaHAsknBA6M2gKszMZXunqFofGXCCk4UCXHMdzd3qtBpndSHuM2JgxBE9M3APyl/DlENt4FEe0C7mJwbcC/ZA==",
```

Comment:
@rynowak Do you happen to know what triggered all the changes here? Did you regenerate the file from scratch?

Since there aren't any `package.json` changes in this PR, it may be that the `package-lock.json` is not needed.

Reply:
I've never really be able to figure out what's going on with this file. I regularly clean/rebuild my repro. 1/20 times I forget and accidentally just include it in a PR.